src/transform-sdk/tests/test_util.go

```diff
@@ -0,0 +1,81 @@
+/*
+* Copyright 2024 Redpanda Data, Inc.
+*
+* Use of this software is governed by the Business Source License
+* included in the file licenses/BSL.md
+*
+* As of the Change Date specified in that file, in accordance with
+* the Business Source License, use of this software will be governed
+* by the Apache License, Version 2.0
+ */
+
+package integration_tests
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/redpanda"
+	"github.com/twmb/franz-go/pkg/kgo"
+)
+
+// Load a Wasm file, if it doesn't exist the test is skipped.
+func loadWasmFile(t *testing.T, envVar string) []byte {
+	wasmFile, ok := os.LookupEnv(envVar)
+	if !ok {
+		t.Skip("Test Wasm file for", envVar, "is missing")
+	}
```

Comment:
nit: don't we risk here that we'll miss something from the build setup and now the tests are passing without checking anything? I'm wondering if this should be guarded with an extra check (like an extra env var "IGNORE_MISSING=true"), that we only set locally.

Reply:
Correct we do risk that, which is why tests are run in verbose mode.

We do want this behavior as not every SDK will always have all the features. I do like the explicit idea, I'll force usage of an explicit environment variable like `IDENTITY=unimplemented` or something.